services/property-svc/migrations/000001_init.up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS select_datas (
+	id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+	allow_freetext boolean NOT NULL DEFAULT false
+);
+
+CREATE TABLE IF NOT EXISTS select_options (
+	id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+	name text NOT NULL,
+	description text NOT NULL DEFAULT '',
+	is_custom boolean NOT NULL DEFAULT false,
+	select_data_id uuid NOT NULL,
+	FOREIGN KEY (select_data_id)
+		REFERENCES select_datas(id)
+		ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS field_type_datas (
```

Comment:
Yes I also thought about that. 
I decided to put that into an extra table because I wanted to model our db representation as close as possible to our event sourced structure, and I think it's not too much of an overhead.
Do you think I should get rid of it? @FoseFx

Reply:
For field types that do not need additional data we would need to maintain a field data entry anyway. Also, we constantly need to `JOIN` the two tables, which takes a lot of computation to do ig.